nodes.py

```diff
@@ -979,43 +979,43 @@ def expand_image(self, image, left, top, right, bottom, feathering):
 
 
 NODE_CLASS_MAPPINGS = {
+    "CLIP Loader": CLIPLoader,
+    "CLIP Set Last Layer": CLIPSetLastLayer,
+    "CLIP Text Encode": CLIPTextEncode,
+    "CLIP Vision Encode": CLIPVisionEncode,
+    "CLIP Vision Loader": CLIPVisionLoader,
+    "Checkpoint Loader": CheckpointLoader,
+    "Checkpoint Loader (Simple)": CheckpointLoaderSimple,
+    "Conditioning Combine": ConditioningCombine,
+    "Conditioning Set Area": ConditioningSetArea,
+    "ControlNet Apply": ControlNetApply,
+    "ControlNet Loader": ControlNetLoader,
+    "Diff ControlNet Loader": DiffControlNetLoader,
+    "Empty Latent Image": EmptyLatentImage,
+    "Image Invert": ImageInvert,
+    "Image Padding (Outpaint)": ImagePadForOutpaint,
+    "Image Scale": ImageScale,
     "KSampler": KSampler,
-    "CheckpointLoader": CheckpointLoader,
-    "CheckpointLoaderSimple": CheckpointLoaderSimple,
-    "CLIPTextEncode": CLIPTextEncode,
-    "CLIPSetLastLayer": CLIPSetLastLayer,
-    "VAEDecode": VAEDecode,
-    "VAEEncode": VAEEncode,
-    "VAEEncodeForInpaint": VAEEncodeForInpaint,
-    "VAELoader": VAELoader,
-    "EmptyLatentImage": EmptyLatentImage,
-    "LatentUpscale": LatentUpscale,
-    "SaveImage": SaveImage,
-    "PreviewImage": PreviewImage,
-    "LoadImage": LoadImage,
-    "LoadImageMask": LoadImageMask,
-    "ImageScale": ImageScale,
-    "ImageInvert": ImageInvert,
-    "ImagePadForOutpaint": ImagePadForOutpaint,
-    "ConditioningCombine": ConditioningCombine,
-    "ConditioningSetArea": ConditioningSetArea,
-    "KSamplerAdvanced": KSamplerAdvanced,
-    "SetLatentNoiseMask": SetLatentNoiseMask,
-    "LatentComposite": LatentComposite,
-    "LatentRotate": LatentRotate,
-    "LatentFlip": LatentFlip,
-    "LatentCrop": LatentCrop,
-    "LoraLoader": LoraLoader,
-    "CLIPLoader": CLIPLoader,
-    "CLIPVisionEncode": CLIPVisionEncode,
-    "StyleModelApply": StyleModelApply,
-    "ControlNetApply": ControlNetApply,
-    "ControlNetLoader": ControlNetLoader,
-    "DiffControlNetLoader": DiffControlNetLoader,
-    "StyleModelLoader": StyleModelLoader,
-    "CLIPVisionLoader": CLIPVisionLoader,
-    "VAEDecodeTiled": VAEDecodeTiled,
-    "VAEEncodeTiled": VAEEncodeTiled,
+    "KSampler (Advanced)": KSamplerAdvanced,
+    "Latent Composite": LatentComposite,
+    "Latent Crop": LatentCrop,
+    "Latent Flip": LatentFlip,
+    "Latent Rotate": LatentRotate,
```

Comment:
This isn't the correct way to rename the nodes and will probably break everything. The correct way would be to add a "friendly name" field that sets this: https://github.com/comfyanonymous/ComfyUI/blob/master/server.py#L156